src/test/unit_tests/jest_tests/test_nc_nsfs_account_cli.test.js

```diff
@@ -103,6 +103,28 @@ describe('manage nsfs cli account flow', () => {
             assert_account(account_symlink, account_options);
         });
 
+        it('should fail - cli create account invalid access_key(leading space in access_key)', async () => {
+            const { type, name, uid, gid, secret_key } = defaults;
+            const account_options = { config_root, name, uid, gid, secret_key};
+            const action = ACTIONS.ADD;
+            const command = create_command(type, action, account_options);
+            const flag = 'access_key';
+            const value = ' 12345678912345678ABC'; // leading space for access_key
+            const res = await exec_manage_cli_add_leading_space_option(command, flag, value);
+            expect(JSON.parse(res.stdout).error.message).toBe(ManageCLIError.InvalidArgument.message);
```

Comment:
If you want you can check the code is the same and not the message (I know that we have checks of message in the code).
* If you fix, then in both cases.

Reply:
fixed, Thanks